code/language/c/loop/do-while.c

```diff
@@ -0,0 +1,20 @@
+/* do-while loop*/
+/*C program to print sum of first 5 natural numbers using do..while loop*/
+/*do- while loop is a loop for performing repetetive tasks. In this loop the body of the loop is executed*/
+/*atleast once, and then the test condition is checked. If it evaluates to true, the loop continues to run otherwise it is exited.*/
+
+#include <stdio.h>
+
+int main() {
+  int sum = 0, i = 1; /*initialization of counter variable i*/
+  do {
+    sum += i;
+    i+=1; /*increment of counter variable*/
```

Comment:
It should be 'i += 1'.
Add the spaces wherever necessary according to our C style guide.